.NET Core/Embed for your organization/UserOwnsData/Startup.cs

```diff
@@ -20,39 +22,42 @@ public class Startup
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
-            Configuration["AzureAd:Instance"] = Configuration["AzureAd:AuthorityUrl"];
         }
 
         public IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddOptions();
-
-            // Graph api base url
-            var graphBaseUrl = "https://graph.microsoft.com/v1.0";
-
-            // Graph scope for reading logged in user's info
-            var userReadScope = "user.read";
-
-            // List of scopes required
-            string[] initialScopes = new string[] { userReadScope };
-
-            services.AddMicrosoftIdentityWebAppAuthentication(Configuration)
-                .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
-                .AddMicrosoftGraph(graphBaseUrl, userReadScope)
-                .AddSessionTokenCaches();
-
+            // Configure session services
+        services.AddSession(options =>
```

Comment:
Remove the options. if we need to add session, simply call services.AddSession()